ui/components/component-library/avatar-with-badge/README.mdx

```diff
@@ -0,0 +1,33 @@
+import { Story, Canvas, ArgsTable } from '@storybook/addon-docs';
+
+import { AvatarWithBadge } from './avatar-with-badge';
+
+# AvatarWithBadge
+
+The `AvatarWithBadge` is a wrapper component that adds badge display options to avatars.
+
+<Canvas>
+  <Story id="ui-components-component-library-avatar-with-badge-avatar-with-badge-stories-js--default-story" />
+</Canvas>
+
+## Props
+
+The `AvatarWithBadge` accepts all props below 
+
+<ArgsTable of={AvatarWithBadge} />
+
+### Badge Position
+
+Use the `badgePosition` prop to set the position of the badge, it can have two values `Top` and `Bottom`
```

Comment:
Add the Badge Position story here

```suggestion
Use the `badgePosition` prop to set the position of the badge, it can have two values `Top` and `Bottom`

<Canvas>
  <Story id="ui-components-component-library-avatar-with-badge-avatar-with-badge-stories-js--badge-position" />
</Canvas>

```

Reply:
Double check that is correct I haven't tested it 😅